devicon.json

```diff
@@ -6632,6 +6632,29 @@
             }
         ]
     },
+    {
+        "name": "neatbeans",
```

Comment:
there is a typo
```suggestion
        "name": "netbeans",
```

Reply:
I for one, think neatbeans is a pretty neat name!